Documentation update for cli graph command (#32262)
hashicorp/terraform: website/docs/cli/commands/graph.mdx

$ terraform graph | dot -Tsvg > graph.svg
```

> Alternatively, `dot` can export to other image file formats by changing the flag input:

Reviewer: This looks great to me with some small tweaks and an additional line between the sentence and the code block:
```suggestion
Alternatively, `dot` can export other image file formats if you change the flag input:

```
